setting-up-multifactor-authentication.md

```diff
@@ -6,13 +6,45 @@ previous: /run-your-own-gem-server
 next: /setting-up-webauthn-mfa
 ---
 
-<em class="t-gray">Want to better protect your rubygems.org account?</em>
+<em class="t-gray">Want to better protect your RubyGems.org account?</em>
 
 Your RubyGems.org account is important! Unauthorized access of your account
 can lead to irrevocable damage to your gem's reputation. We highly recommend
 that you enable MFA for both UI and API. When enabled, this will mean that
-you need to use MFA for signing into rubygems.org and when running `gem signin`,
+you need to use MFA for signing into RubyGems.org and when running `gem signin`,
 `push`, `owner --add`, `owner --remove` and `yank`.
 
 You may enable MFA using [WebAuthn](/setting-up-webauthn-mfa) or by
 using [one-time passwords (OTP)](/setting-up-otp-mfa).
+
+## Authentication levels
+
+When you register a new device or enable MFA for the first time, we will enable
+MFA for both the UI and the API. If you go to the "Edit Settings" page again, in the "Multi-factor Authentication" section, you
+will see a dropdown menu with these options:
+
+- **UI only (Deprecated)**: sign in from browser, updating MFA levels and resetting password will require OTP code. These are referred to as UI operations.
```

Comment:
ui_only isn't in the dropdown anymore:
<img width="661" alt="Screenshot 2023-07-20 at 3 06 27 PM" src="https://github.com/rubygems/guides/assets/71022385/9d67755a-86c6-4663-8e02-45c2483a6cf0">

Reply:
I was a bit torn on this. `ui_only` can still show up if the user's current level is `ui_only` https://github.com/rubygems/rubygems.org/blob/20f7677f4798f66403971407768b72a11304a431/app/controllers/settings_controller.rb

However, no one should be on the level since we ran a rake task. I still prefer to remove it when the clean up to remove `ui_only` in the codebase is done (which has been on my TODOs for a while now 😅 )